transition-scripts/deploy-gold-in-standby.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+usage() {
+    cat <<EOF
+Deploy Keycloak resources in the target Gold namespaces in standby mode.
+This is for use in the case that gold was not able to be put into standby mode.
+The user will have to scale Patroni-Gold pods to zero before running. This script will
+delete existing patroni PVCs and configmaps.
+
+Usages:
+    $0 <namespace>
+
+Available namespaces:
+    - c6af30-dev
+    - eb75ad-dev
+    - eb75ad-test
+    - eb75ad-prod
+
+Pre-conditions:
+    - the patroni cluster in Golddr must be healthy and in active mode.
+    - the Gold patroni pods are scaled to zero.
```

Comment:
I think this pre-condition can be removed as we might want to run this script when patroni pods are running but not able to sync with golddr.

Reply:
The reason I have the 'scale to zero' is because the clean up script hangs  when it tries to delete the PVCs.  The status of the PVCs show's as 'Terminating' but it seems to hang.  Scaling the patroni pods to zero prevents this.